accelerator/BUILD

```diff
@@ -5,7 +5,10 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_push")
 
 cc_binary(
     name = "accelerator",
-    srcs = select({
+    srcs = [
+        "config.c",
+        "config.h",
```

Comment:
Should `config.h` be listed as well?

Reply:
I tried it. It works fine without `config.h`. 
However, there are some variables are guarded with `#ifdef MQTT_ENABLE `, so I am afraid that we will meet other bugs if we removed `config.h`